src/style.css

```diff
@@ -0,0 +1,145 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+/* font-family: 'Roboto', sans-serif; */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.img {
+  padding: 29px 0 41px;
+}
+
+.img__product {
+  width: 160px;
+  height: 134px;
+}
+
+.content {
+  margin: 0 14px 14px;
+}
+
+.content__title {
+  padding: 0 0 6px;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+}
+
+.content__code {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  padding: 0 0 15px;
+  margin: 0;
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 0 20px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.stars :nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars :last-child {
+  background-image: url(images/star.svg);
+  margin: 0;
+}
+
+.span {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  color: #000;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding: 0 0 20px;
+}
+
+.price :nth-child(n) {
+  margin: 0;
+}
+
+.price__title {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  color: #000;
+}
+
+.price__cost {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+/* .buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+} */
+
+.buy {
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  background-color: #00acdc;
+  border-radius: 5px;
+  min-width: 170px;
+  min-height: 40px;
```

Comment:
You can use just 'width' and 'height' instead of 'min-width' and 'min-height' here